### PR TITLE
pls ignore thanks

### PR DESCRIPTION
**Pull request by gloorgon**

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -139,12 +162,14 @@ Discord.on('message', function(message) {
 });
 
 Discord.on('message', function(message) {
+	if(!message.author.bot) return
```
Comment (tonkku107) [inline bot.js]: nvm

File: bot.js [hunk 2]
```diff
@@ -139,12 +162,14 @@ Discord.on('message', function(message) {
...
 
 Discord.on('message', function(message) {
+	if(!message.author.bot) return
 	if (message.content.startsWith('!say ')) {
 		return message.reply(message.content.substring('!say '.length));
 	}
 })
 
 Discord.on('message', function (message) {
+	if(!message.author.bot) return
```
Comment (tonkku107) [inline bot.js]: Are we there yet?

File: bot.js [hunk 3]
```diff
@@ -153,6 +178,7 @@ Discord.on('message', function (message) {
 })
 
 Discord.on('message', function (message) {
+	if(!message.author.bot) return
```
Comment (tonkku107) [inline bot.js]: Almost

File: bot.js [hunk 4]
```diff
@@ -163,6 +189,7 @@ Discord.on('message', function (message) {
 
 const guildMemberAdd = 'message';
 Discord.on(guildMemberAdd, function (aaaaaaAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA)                      {
+	if(!message.author.bot) return
```
Comment (tonkku107) [inline bot.js]: Finally

**Comment by gloorgon** [comment]
epic

**Comment by gloorgon** [comment]
@tonkku107 hello sir i believe i deserver my nobel prize now please and thank you

File: bot.js [hunk 5]
```diff
@@ -45,26 +58,33 @@ Discord.on('message', function (message) {
 		return message.reply('Hello!');
 	  }
   });
+var i = "cat"
 
 Discord.on('message',  function (message) {
+	if(message) return message.reply(i)
```
Comment (tonkku107) [inline bot.js]: I don't like this part

File: bot.js [hunk 6]
```diff
@@ -32,39 +37,53 @@ Discord.on(	'guildMemberAdd'	, function		(member) {
...
 });
 
   Discord.on('message', function (message) {
+	  if(!message.author.bot) return
 	if (message.content === 'sa') {
 		return message.reply('Aleyküm selam.');
 	  }
   			});
   		Discord.on('message', function (message) {
+			if(!message.author.bot) return
```
[the remark this answers is not in the document]
Comment (tonkku107) [reply bot.js]: This still exists

File: bot.js [hunk 7]
```diff
@@ -118,6 +139,7 @@ Discord.on('message', function (message) {
 });
 
 Discord.on('message', 	function(message) {
+	if(!message.author.bot) return
```
[the remark this answers is not in the document]
Comment (tonkku107) [reply bot.js]: This one too

File: bot.js [hunk 8]
```diff
@@ -139,39 +158,50 @@ Discord.on('message', function(message) {
 });
 
 Discord.on('message', function(message) {
+	if(!message.author.bot) {
+		return message.reply("hi")
```
Comment (tonkku107) [inline bot.js]: Why don't you make a separate event for this

File: bot.js [hunk 9]
```diff
@@ -139,39 +158,50 @@ Discord.on('message', function(message) {
...
 	if (message.content.startsWith('!say ')) {
 		return message.reply(message.content.substring('!say '.length));
 	}
 })
 
 Discord.on('message', function (message) {
-		if (message.content.startsWith('!randomcat') && true){
+	
+if (message.content.startsWith('!randomcat') && true){
+	 else if(!message.author.bot) {
```
Comment (tonkku107) [inline bot.js]: I dont like this either

File: bot.js (hunk 4 above)
Comment (tonkku107) [reply bot.js]: This one exists too

File: bot.js [hunk 10]
```diff
@@ -139,39 +158,50 @@ Discord.on('message', function(message) {
...
-
+var e = "message"
+var ge = e
+var age = ge
+var sage = age
+var ssage = sage
+var essage = ssage
+var message = essage
+Discord.on(message, function         (hi) { 
+	hi.reply("i").then(bye_cry_emoji => { bye_cry_emoji.reply("hi") }) })
```
Comment (tonkku107) [inline bot.js]: Literally what the fuck

**Comment by gloorgon** [comment]
wait

**Comment by gloorgon** [comment]
this is shit

**Comment by gloorgon** [comment]
hold on

**Pull request closed without merging**